internal/resolver/mtaresolver_test.go

```diff
@@ -71,8 +71,12 @@ func getExpected(expected []string) string {
 	return <-out
 }
 
-func callResolveAndValidateOutput(wd, moduleName, yamlPath string, expected []string) {
-	actualStr := callResolveAndGetOutput(wd, moduleName, yamlPath)
+func callResolveAndValidateOutput(wd, moduleName, yamlPath string, expected []string, envFileNameOps ...string) {
```

Comment:
Same as above - you only use the first value, so the parameter `envFileNameOps` should not be multi-valued

Reply:
Done